tests/suites/test_suite_md.function

```diff
@@ -304,13 +305,14 @@ void mbedtls_md_hmac( char *text_md_name, int trunc_size,
     TEST_ASSERT ( mbedtls_md_hmac( md_info, key_str->x, key_str->len,
                                    src_str->x, src_str->len, output ) == 0 );
 
-    TEST_ASSERT( hexcmp( output, hex_hash_string->x, trunc_size,
+    TEST_ASSERT( trunc_size == hex_hash_string->len );
+    TEST_ASSERT( memcmp( output, hex_hash_string->x,
                          hex_hash_string->len ) == 0 );
 }
 /* END_CASE */
 
 /* BEGIN_CASE */
-void md_hmac_multi( char *text_md_name, int trunc_size, data_t *key_str,
+void md_hmac_multi( char *text_md_name, unsigned int trunc_size, data_t *key_str,
```

Comment:
This should stay as `int`.

Reply:
You right. I fix it to use cast instead.